jetstream/cli.py

```diff
@@ -188,7 +211,35 @@ def execute(
         ] = ExternalConfigCollection.from_github_repo,
         today: Optional[datetime] = None,
     ) -> bool:
+        run_experiments = self._experiments_to_analyse(experiment_getter)
+        worklist = []
+
+        for experiment in run_experiments:
+            if self.date == All:
```

Comment:
We spell this as `isinstance(self.date, AllType)` in some other places, but I think this is actually equally correct and also more legible; can you convert those other uses to this form so we're consistent?

Reply:
Looks like we used `isinstance(self.date, AllType)` at these other places to make mypy happy